yarn.lock

```diff
@@ -53,18 +62,38 @@
     "@babel/template" "7.0.0-beta.44"
     "@babel/types" "7.0.0-beta.44"
 
+"@babel/helper-function-name@^7.1.0":
+  version "7.1.0"
```

Comment:
For all of these it seems like we're pointing to the minor version, but we still have a block pointing to the beta version, i.e. https://github.com/jwplayer/jwplayer/pull/3237/files#diff-8ee2343978836a779dc9f8d6b794c3b2R58
is that safe ?

Reply:
You don't need to worry about changes in the lock file. Especially for child dependencies like this.

Reply:
What you do need to do @DanFerrer is rebase and run `yarn install` on your branch. The branch and the lock file are out of date.